src/analyses/libraryFunctions.ml

```diff
@@ -88,6 +96,43 @@ let posix_descs_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("getsockopt", unknown [drop "sockfd" []; drop "level" []; drop "optname" []; drop "optval" [w]; drop "optlen" [w]]);
     ("gethostbyaddr", unknown [drop "addr" [r_deep]; drop "len" []; drop "type" []]);
     ("gethostbyaddr_r", unknown [drop "addr" [r_deep]; drop "len" []; drop "type" []; drop "ret" [w_deep]; drop "buf" [w]; drop "buflen" []; drop "result" [w]; drop "h_errnop" [w]]);
+    ("sigaction", unknown [drop "signum" []; drop "act" [r_deep; s_deep]; drop "oldact" [w_deep]]);
+    ("tcgetattr", unknown [drop "fd" []; drop "termios_p" [r_deep]]);
+    ("tcsetattr", unknown [drop "fd" []; drop "optional_actions" []; drop "termios_p" [w_deep]]);
+    ("access", unknown [drop "pathname" [r]; drop "mode" []]);
+    ("ttyname", unknown [drop "fd" []]);
+    ("shm_open", unknown [drop "name" [r]; drop "oflag" []; drop "mode" []]);
+    ("sched_get_priority_max", unknown [drop "policy" []]);
+    ("mprotect", unknown [drop "addr" []; drop "len" []; drop "prot" []]);
+    ("ftime", unknown [drop "tp" [w]]);
+    ("timer_create", unknown [drop "clockid" []; drop "sevp" [r; w; s]; drop "timerid" [w]]);
+    ("timer_settime", unknown [drop "timerid" []; drop "flags" []; drop "new_value" [r_deep]; drop "old_value" [w_deep]]);
+    ("timer_gettime", unknown [drop "timerid" []; drop "curr_value" [w_deep]]);
+    ("timer_getoverrun", unknown [drop "timerid" []]);
+    ("lstat", unknown [drop "pathname" [r]; drop "statbuf" [w]]);
+    ("getpwnam", unknown [drop "name" [r]]);
+    ("strndup", unknown [drop "s" [r]; drop "n" []]);
+    ("freeaddrinfo", unknown [drop "res" [f_deep]]);
+    ("getgid", unknown []);
+    ("pselect", unknown [drop "nfds" []; drop "readdfs" [r]; drop "writedfs" [r]; drop "exceptfds" [r]; drop "timeout" [r]; drop "sigmask" [r]]);
+    ("strncasecmp", unknown [drop "s1" [r]; drop "s2" [r]; drop "n" []]);
+    ("getnameinfo", unknown [drop "addr" [r_deep]; drop "addrlen" []; drop "host" [w]; drop "hostlen" []; drop "serv" [w]; drop "servlen" []; drop "flags" []]);
+    ("strtok_r", unknown [drop "str" [r; w]; drop "delim" [r]; drop "saveptr" [w]]);
+    ("kill", unknown [drop "pid" []; drop "sig" []]);
+    ("closelog", unknown []);
+    ("dirname", unknown [drop "path" [r]]);
+    ("setpgid", unknown [drop "pid" []; drop "pgid" []]);
+    ("dup2", unknown [drop "oldfd" []; drop "newfd" []]);
+    ("pclose", unknown [drop "stream" [w; f]]);
+    ("getcwd", unknown [drop "buf" [w]; drop "size" []]);
+    ("inet_pton", unknown [drop "af" []; drop "src" [r]; drop "dst" [w]]);
+    ("inet_ntop", unknown [drop "af" []; drop "src" [r]; drop "dst" [w]; drop "size" []]);
+    ("inet_aton", unknown [drop "cp" [r]; drop "inp" [w]]);
```

Comment:
True, I think I just put it here together with the others, but I'll move it to glibc since it's listed here: https://www.gnu.org/software/gnulib/manual/html_node/Glibc-arpa_002finet_002eh.html.